src/webview/mod.rs

```diff
@@ -695,6 +697,14 @@ pub trait WebViewBuilderExtWindows {
   ///
   /// Defaults to [`Theme::Auto`] which will follow the OS defaults.
   fn with_theme(self, theme: Theme) -> Self;
+
+  /// Determines whether the custom protocols should use `http://<scheme>.localhost` instead of `http://<scheme>.localhost`.
```

Comment:
```suggestion
  /// Determines whether the custom protocols should use `http://<scheme>.localhost` instead of the default `https://<scheme>.localhost`.
```